src/main/java/io/kestra/plugin/notifications/sentry/SentryExecution.java

```diff
@@ -0,0 +1,80 @@
+package io.kestra.plugin.notifications.sentry;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.tasks.VoidOutput;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.notifications.ExecutionInterface;
+import io.kestra.plugin.notifications.services.ExecutionService;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Send a Sentry message with the execution information",
+    description = "The message will include a link to the execution page in the UI along with the execution ID, namespace, flow name, the start date, duration and the final status of the execution, and (if failed) the task that led to a failure.\n\n" +
+        "Use this notification task only in a flow that has a [Flow trigger](https://kestra.io/docs/administrator-guide/monitoring#alerting). Don't use this notification task in `errors` tasks. Instead, for `errors` tasks, use the [SentryAlert](https://kestra.io/plugins/plugin-notifications/tasks/sentry/io.kestra.plugin.notifications.sentry.sentryalert) task."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Send a Sentry notification on a failed flow execution",
+            full = true,
+            code = """
+                id: failure_alert
+                namespace: prod.monitoring
+
+                tasks:
+                  - id: send_alert
+                    type: io.kestra.plugin.notifications.sentry.SentryExecution
+                    url: "{{ secret('SENTRY_ALERT') }}" # format: https://{HOST/URI}/api/{PROJECT_ID}/store/?sentry_version=7&sentry_client=java&sentry_key={PUBLIC_KEY}
+                    eventId: "fc6d8c0c43fc4630ad850ee518f1b9d1"
+                    timestamp: "2023-05-02T17:41:36Z"
+                    platform: "java"
+                    level: "error"
+                    transaction: "/execution/id/321312"
+                    serverName: "localhost:8080"
+                    extra:
+                      Namespace: {{execution.namespace}}
+                      Flow ID: {{execution.flowId}}
+                      Execution ID: {{execution.id}}
+                      Execution Status: {{execution.state.current}}
+                      Link: {{link}}
```

Comment:
the same changes as before would be needed to access proper template values:

![image](https://github.com/kestra-io/plugin-notifications/assets/86264395/5a02bc55-eb8c-44cd-a54f-d4a9a61100cf)

![image](https://github.com/kestra-io/plugin-notifications/assets/86264395/90d34fa0-3868-42a3-8cbd-200006402a4c)